src/components/Menu/index.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+import { FaBars, FaTimes } from "react-icons/fa";
+import { Menu, NavLinks, MobileIcon, NavMenu } from "./Menu";
+import logo from "../../Images/Logo.svg";
+
+const Navbar = () => {
+  const [open, setOpen] = useState(false);
+
+  const menuToggle = () => {
```

Comment:
Se preferir pode fazer essa função em uma linha só: 
```suggestion
const menuToggle = () => setOpen(!open);
```